pkg/cluster/kubernetes/talos_managed.go

```diff
@@ -126,6 +126,20 @@ func UpgradeTalosManaged(ctx context.Context, cluster UpgradeProvider, options U
 
 	options.Log("discovered controlplane nodes %q", options.controlPlaneNodes)
 
+	talosClient, err := cluster.Client()
```

Comment:
was this intended? the PR description misses this?

Reply:
it was a typo in the PR list, yes, I intended to backport the new checks you implemented!